src/types.js

```diff
@@ -125,6 +125,8 @@ export type Manifest = {
   main?: string,
 
   workspaces?: Array<string>,
+
+  fresh?: boolean
```

Comment:
Could you add a small comment here to explain what means `fresh` in this context, and a short summary detailing how it is used inside the Yarn internals?